src/compas/tolerance.py

```diff
@@ -108,8 +114,9 @@ class Tolerance(Data):
     """
 
     def __new__(cls, *args, **kwargs):
-        if not hasattr(cls, "_instance"):
-            cls._instance = super(Tolerance, cls).__new__(cls)
+        if not isinstance(cls._instance, cls):
```

Comment:
trying to understand this one... couldn't this be `if cls_instance is not None`? Or there are some other possible type for the instance?